README.md

```diff
@@ -1,28 +1,279 @@
-# XiSearch
-Actually a library containing several tools - most prominently a cross-link MS search engine.
+## xiSEARCH 1.7.6.7 documentation
 
-XiSearch is implemented as a Java Application. Therefore it requires that java is installed. We recommend to use the the latest update of JAVA version 8.
 
+XiSearch is implemented as a Java Application. Therefore it requires that java is installed. We recommend to use the the latest update of JAVA version 8 or above. 
+The latest version as binary can be downloaded from https://rappsilberlab.org/downloads/
 
-The latest version as binary can be downloaded from
-https://rappsilberlab.org/downloads/
+### Background
+xiSEARCH is a search engine for crosslinking mass spectrometry (crosslinking MS). It supports data acquired with ThermoFisher Orbitrap instruments (.raw format) that have been converted to peak. files (.mgf format), for example with [ProteoWizard MsConvert](https://proteowizard.sourceforge.io/). It then searches the peakfiles against a sequence database in .fasta format to identify crosslinked peptide pairs from mass spectra. The search algorithm uses a target-decoy approach outlined in XXX, which enables false-discovery rate (FDR) estimation. The FDR calculation on the xiSEARCH result is performed by xiFDR.
 
-This includes a startup script for the XiSearch gui.
+xiSEARCH is a flexible search engine that allows for extensive configuration of the search options and of the search scoring methods in crosslink identification. Nevertheless, its design suits best data acquired at high resolution in MS1 and MS2- in the Rappsilber lab, we acquire with 120k resolution in ms1 and 60k resolution ms2. Currently, xiSEARCH does not support MS3 approaches.
 
+The xiSEARCH algorithm is described in detail in [Mendez, Fischer *et al.* Mol. Sys. Bio. 2019](https://www.embopress.org/doi/full/10.15252/msb.20198994). The xiSEARCH scoring function is made up of several terms accounting for the goodness of fit of the spectra to the peptide pair selected from the database, including fragment mass error, percentage intensity explained, number of fragments, number of crosslinked fragments.
 
-the Gui provides several tabs to configure your search. 
-In the first tab (Run) the result file (result) and optionally a for writing out full annotations of the spectra (peak annotations) can be defined.
+Scoring happens in three stages: 
+
+1. alpha candidates are selected and scored
+2. top n aplpha candidates are taken and all matching beta-candidates will be selected and prescored
+3. the top X of these are then fully matched and scored
+
+The scoring function is applied to explain each spectrum with a target-target pair, a target-decoy pair, a decoy-decoy pair, a linear target sequence or a linear decoy sequence. Error control by false discovery rate estimation is then performed in a separate step with xiFDR.
```

Comment:
at this point xisearch does not care about target or decoy - so it could well be that only decoys are considered or only targets.

Reply:
Actually it does the full scoring on the top-x alpha-beta-candidates + all possible linear matches

Reply:
then the same should be changed in the default config, which is where this text comes from. We should also include that figure i mentioned here in the readme...

Reply:
Not sure where you found that text - as for the figure as it is a pull request from your fork - ithink you probably need to add teh file: ![xi_search_strategy](https://user-images.githubusercontent.com/6330440/201888945-7f9bafbb-7b96-47e0-8e22-919f6bec7c7e.jpg)

Reply:
right it is not from there... i remember copy pasting that, though it does read similar to the legend of https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6753376/ fig EV6.